buildkite/src/Command/Base.dhall

```diff
@@ -70,13 +87,16 @@ let Config =
       , docker : Optional Docker.Type
       , docker_login : Optional DockerLogin.Type
       , summon : Optional Summon.Type
+      , retry : Optional Retry.Type
```

Comment:
Worth updating to allow for a list of Retry Types? So that operators don't have to make the choice to support one retry spec vs. another (especially considering the Infra spec added by default and probably a good idea to always have).

Reply:
Looking into this, you're right it's quite difficult, but I think I can make it work